engine/lite/k8sDeploymentManager/src/main/scala/pl/touk/nussknacker/k8s/manager/deployment/DeploymentPreparer.scala

```diff
@@ -55,6 +58,17 @@ class DeploymentPreparer(config: K8sDeploymentManagerConfig) extends LazyLogging
     deploymentLens(userConfigurationBasedDeployment)
   }
 
+  private def deploymentStrategy(canonicalProcess: CanonicalProcess): Strategy = {
+    canonicalProcess.metaData.typeSpecificData match {
+      case data: ScenarioSpecificData => data match {
+        case _: LiteStreamMetaData => Deployment.Strategy.Recreate
+        case _: RequestResponseMetaData => Deployment.Strategy(rollingUpdate = Deployment.RollingUpdate())
+        case _: StreamMetaData => throw new IllegalStateException("Deploying flink streaming unsupported on k8s")
```

Comment:
maybe just handle `_` as all other unexpected types like it is done in other places (for example for initial meta data)?

Reply:
done